src/k8s-engine/images/imageFactory.ts

```diff
@@ -0,0 +1,21 @@
+import { ImageProcessor } from '@/k8s-engine/images/imageProcessor';
+import NerdctlImageProcessor from '@/k8s-engine/images/nerdctlImageProcessor';
+import * as K8s from '~/k8s-engine/k8s';
```

Comment:
We seem to use `@` over `~` for some reason. No, I have no idea _what_ the reason is… But at least all the imports should match.

Reply:
My IDE (JetBrains...) uses that notation when it adds a needed import statement. I missed that one.